tests/app/views/fixtures/test_assessment.py

```diff
@@ -0,0 +1,59 @@
+import json
+
```

Comment:
this test folder hierarchy is getting pretty deep.. maybe divide it up at top level?

old vs new or two similar names. put this in new and tests can be moved from old to new as they are gradually cleaned up